api/Document.json

```diff
@@ -5849,9 +5849,17 @@
             "oculus": "mirror",
             "opera": "mirror",
             "opera_android": "mirror",
-            "safari": {
-              "version_added": "17.4"
-            },
+            "safari": [
+              {
+                "version_added": "26"
+              },
+              {
+                "version_added": "17.4",
+                "version_removed": "26",
+                "partial_implementation": true,
+                "notes": "Won't work when parsed HTML contains web components ([bug 296420](https://bugs.webkit.org/show_bug.cgi?id=296420).)"
```

Comment:
I think this should use impl_url rather than be a bug link in the note. (Someone else can confirm though)